byon/files/seaclouds-catalog.bom

```diff
@@ -186,13 +186,14 @@ brooklyn.catalog:
       type: org.apache.brooklyn.entity.brooklynnode.BrooklynNode
       id: deployer
       name: Deployer
-      version: 0.8.0-incubating
+      version: 0.9.0-SNAPSHOT
```

Comment:
I agree with you, @andreaturli. However, the Deployer needs 0.9.0 brooklyn version to manage brooklyn-tosca project. So, I thought we could use SNAPSHOT version as a temporal solution.